src/reference/asciidoc/whats-new.adoc

```diff
@@ -16,3 +16,8 @@ When using <<receiving-batch,batching>>, `@RabbitListener` methods can now recei
 
 The Log4J and Logback `AmqpAppender` s now support a `verifyHostname` SSL option.
 See <<logging>> for more information.
+
+===== MessageListenerAdapter Changes
+
+You can now receive Channel and Message objects in listener by means of doing extends MessageListenerAdapter.
```

Comment:
I would just say that `MessageListenerAdapter` provides now a new method and an old one is deprecated.

What you say here is just true even now.

The `MessageListenerAdapter` must be wrapped to the code, too.